pupil_src/shared_modules/video_capture/utils.py

```diff
@@ -10,8 +10,52 @@
 '''
 
 import numpy as np
+import cv2
 
 
+class GetExposureTime(object):
```

Comment:
Even though [PEP8 would agree with this naming](https://www.python.org/dev/peps/pep-0008/#class-names) it would be more consistent with all other class names in this project to rename this function to `Get_Exposure_Time`.

Additionally, I would like to propose to rename this to simply `Exposure_Time`. Together with the comments below this would yield this naming:

```python
preferred_exposure_time = Exposure_Time(max_ET, frame_rate)
target = preferred_exposure_time.calculate_based_on_frame(frame)
```